cyto_dl/train.py

```diff
@@ -94,6 +95,14 @@ def train(cfg: DictConfig) -> Tuple[dict, dict]:
 
     if cfg.get("train"):
         log.info("Starting training!")
+
+        if cfg.get("weights_only"):
+            # load model from state dict to get around trainer.max_epochs limit, useful for resuming model training from existing weights
+            ckpt_path = cfg.get("ckpt_path")
+            state_dict = torch.load(ckpt_path)["state_dict"]
```

Comment:
Is there any logic validating that when `weights_only`, the other keys need to exist?